R/demo-crowding.R

```diff
@@ -69,6 +70,10 @@ demo_crowding <- function(area, areatype, crowding, race) {
       output = 1
     ) %>%
     req_perform()
+  
+  if (httr2::resp_content_type(resp) != "text/csv") {
+    cli_abort("Invalid input, please check documentation for valid arguments.")
+  }
```

Comment:
I thought you were going to add this to the `process_*` family of functions? I would recommend that, it avoids a bunch of code duplication, especially once we consolidate those functions.